package/rules/url.js

```diff
@@ -0,0 +1,13 @@
+const assetHost = require('../asset_host')
+
+module.exports = {
```

Comment:
@gauravtiwari How can I reference small static images in Rails view template after using this url loader?

Reply:
All images under 10KB would use url loader by default, this includes png, jpg, gif and bmp

Reply:
Then the url loader will break images under 10KB in projects with server rendering